README.md

```diff
@@ -98,25 +99,26 @@ The protobuf extension makes both exporters more performant. _Note: there are so
 ## Installation
 The recommended way to install the library is through [Composer](http://getcomposer.org):
 
-1.)  Install the composer package using [Composer's installation instructions](https://getcomposer.org/doc/00-intromd#installation-linux-unix-macos).
+1)  Install the composer package using [Composer's installation instructions](https://getcomposer.org/doc/00-intromd#installation-linux-unix-macos).
 
-2.)  Add
+
+2)  Add
 ```bash
  "minimum-stability": "dev"
 ```
 
 To your project's `composer.json` file, as this utility has not reached a stable release status yet.
 
-2.)  Install the dependency with composer:
+3) Install the dependency with composer:
 
 ```bash
 $ composer require open-telemetry/opentelemetry
 ```
 
 ## Development
-We use `docker` and `docker-compose` to perform a lot of our static analysis and testing.  
+For repeatability and consistency across different operating systems, we use the [3 Musketeers pattern](https://3musketeers.io/). If you're on Windows, it might be a good idea to use Git bash for following the steps below.
 
-If you're planning to develop for this library, it'll help to install `docker engine` and `docker-compose`.  
+We use `docker` and `docker-compose` to perform a lot of our static analysis and testing. If you're planning to develop for this library, it'll help to install `docker engine` and `docker-compose`.
 
 You can find installation instructions for these packages can be found [here](https://docs.docker.com/install/), under the `Docker Engine` and `Docker Compose` submenus respectively.
```

Comment:
Could you please also remove the redundant "find/found"?
Also these are not necessarily "packages". I think the word "tools" would fit better.